# Phylogenetic Biology - Final Project

# Comparative Phylogeny of Viburnum

## Introduction and Goals
My project is interested in determining if there are differenes in the rate of evolution across the Viburnum phylogeny related to different leaf anatomy and habitat. Viburnum has ancestral H palisade cells which appear with additional arms that increase the surface area. I cells, the traditional columnar palisade cell, have evolved more than once apparently to respond to more direct light. H cells and I cells represent two specific strategies that Viburnum has adopted for light scavening with effects on the overall fitness of the plant. Previous phylogenies of Viburnum have already shown that the lineage has a history of moving out of biomes with biome-specific traits following the transition (Landis et al, 2020). Rates of molecular evolution have been attempted to be linked to life history strategies, but there has not been a consensus (Smith and Donoghue, 2008). Slower carbon accumulation in shade could result in longer generation times and slower relative rates of evolution compared to sun adapted species. 

Working alongside Aleca, I plan to construct an initial maximum liklihood phylogeny using the RADSeq data from the Landis et al project overlayed with H and I cell information. Aleca will be working to determine the phylogenetic independent contrasts for Viburnum, which I plan to help with. Following this, I will use the liklihood approach described Adams to compare the rate of evolutionary change for phenotypic traits. I think this method would also allow me to compare whether there are differential rates of evolution between sun and shade habitats. I would also like to explore how different molecular clocks can impact branch lengths of the phylogeny. To determine this, I would clamp internal nodes to fossil dates, geological events such as Laurasia, and substitution rates and compare the results using a distance measure. 

H and I cell data has been collected by Aleca and other members of the Brodersen lab. RADSeq data is available on Dryad.

## Methods
Phylogenetic Inference
Bayesian Tree Analysis: Using the mcmc_GTR.Rev script, a Bayesian tree was produced from the 173 species and 10 gene matrices. Gene matrices were uploaded to Dryad by Landis et al. Data was stored in nexus file types, which require additional work to remove individual genes from. The script was based on Michael Landis' mol_model.Rev script uploaded in a slightly edited form here. One of the other attempts is also saved in a seperate .rev file.  
Maximum Likelihood Tree: Using the ITS gene from the Viburnum data (saved as a seperate fasta file), a ML and IQTree were produced. These follow the most simplistic GTR script.

Comparative Analysis
Trait data was collected from Schmeler et al and previously collected data from the Edwards lab. 
Using the R packages mvMORPH and phytools, the output of the Bayesian analysis was was pruned and made ultrametric. Using the make.simmap function, the tree was annotated with H and I cell designations including a proposed ancestral state reconstruction. This simmap version of the tree was then used in the mvMORPH function mvOU, which applies an Orstein-Uhlenbeck process to at least two different traits. This was used on as many traits as possible, pairing traits that should share evolutionary processes to ensure reliable inferences. This output theta, alpha, and sigma for H and I cell types which represent ancestral state, selection, and drift respectively. This information would be plotted to determine which traits were evolving most quickly and whether specific traits were more closely linked to H or I cell lineages.
To specifically consider whether specific traits aligned with linages or if there was contingent evolution occuring, ancestral state reconstructions were performed for three sets of traits. These were traits with data for 77 species, traits with data only for H cell type species, and traits with data only for a subset of species related to hydraulics. Seperate pruned trees were created for each trait set, and the function fastAnc was used to create ancestral state reconstructions. 
Phylogenetic Independent Contrasts and Phylogenetic General Least Squares were performed on a subset of the traits to determine if there were strong relationships between them. Many of the hydraulic traits were too similar or were calculated using one another which prevented their inclusion.
Felsenstein's Threshold model was used to find correlation between traits, specifically Vein length density and palisade layer type. The settings were as follows: sample 500, number of generations 200,000, burnin .2*number of generations. 

## Results
H_I_plot is the final version of the simmap vibrunum tree including all species that had trait data available. This newer version resulted in significantly fewer changes between H and I cell types in modern species, which had important consequences for the mvMORPH analysis. Specifically, there are several more traits that had higher alpha values than layer type in the final version. These include vein length density of minor veins, leaf thickness, intercellular air space total, H cell arm length, leaf mass area, H cell arm number, tertiary vein width, xylem diameter, adaxial epidermis thickness, and relative water content at turgor loss. These range across trait types. Cell surface area and cell volume had extremely high sigma values even though their model was considered reliable. This could be related to the difficulty in measuring these traits or plasticity.
The final I/H theta ratio also changed. Previous issues with H cell type data were resolved which firmly placed them as traits which were larger ancestraly for H lineages. Traits that were above 1.25 were coded as I and traits below .75 were coded as H. Xylem area and cell volume were also larger in H lineages. Palisade surface area to volume ratio was only .82. Trichome density and palisade thickness were much larger in I lineages, and the palisade area to mesophyll area ratio, vein lenth density of minor veins, and Huber value were all at 1.25. 
The ancestral character state analyses are labeled according the their trait. Currently, there did not seem to be any contingent evolution patterns when visually comparing the trees. The only possible one would be intercellular air space predating changes in layer type and surface area to volume ratio. Given a more quantative analysis, perhaps additional information could be determined. Traits that are directly connected, like leaf thickness and adaxial thickness, see changes at the same time and place in the tree. H arm length and H arm number seem to be inversely related, such that long armed species have fewer numbers of arms.
PIC and PGLS analysis foudn the obvious significant relationship between photosynthetic max and layer type. It also found that intercellular air space and chloroplast number were not significantly related to layer type, but leaf dimensions were. Additionally, osmotic potential at full turgor and vein length density total were also significantly correlated with layer type. PGLS detected that relative water content at turgor loss was significantly correlated with photosynthesis and surface area to volume ratio but not layer type. 
The threshold coorelation test between vein length density total and layer type had a value of 0.6449364. It produced a posterior density and log likelihood plots which appear as threshold plot in plots. The posterior density plot has a long tail from 0 up to .5 and a well mixed log likelihood.
Additional tests, such as the phylomorphospace graph and PCA were also performed. The phylomorphospace indicates that there is strong clustering in trait space between photosynthetic max and surface area to volume ratio; however, there are still numerous species on the fringes. PCAs indicated that for hydraulic traits, 1 component would be sufficient while 2 would be more effective when considering non hydraulic, non H cell, traits for which there were not NAs. 
## Discussion

## References
https://academic.oup.com/sysbio/article/62/2/181/1667814
https://science.sciencemag.org/content/322/5898/86.full
https://royalsocietypublishing.org/doi/suppl/10.1098/rspb.2012.1110
https://onlinelibrary.wiley.com/doi/full/10.1111/evo.14030?casa_token=mIqnLBIK4p8AAAAA%3AJS_JNc03daK8jGD8bX4mB5OLSzO0QC2V8Bus3D2kfhY9s83RUtJnYpmf-rKSBeSzYJZt6SaJk0Dqo1F3
